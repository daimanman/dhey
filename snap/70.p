{"C":1,"Duration":246352896,"EndTime":1537232386185657600,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537232385939304700,"Status":1,"TaskId":70,"Type":"FILE","Url":"http://192.168.1.192:9091/api/v1/oss","Z":""}