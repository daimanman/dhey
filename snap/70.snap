{"AvgTotal":0.2453436,"Fastest":0.2453436,"Slowest":0.2453436,"Average":0.2453436,"Rps":4.075916388281578,"AvgConn":0.0009967,"AvgDNS":0,"AvgReq":0,"AvgRes":0.0009973,"AvgDelay":0.195478,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0.0009973,"DelayMax":0.195478,"DelayMin":0.195478,"Lats":[0.2453436],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0.0009973],"DelayLats":[0.195478],"Total":245343600,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.2453436,"Count":1,"Frequentcy":1},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0},{"Mark":0.2453436,"Count":0,"Frequentcy":0}],"TaskId":70,"Duration":0.2453436}